documentation/interpretation_guide.md

```diff
@@ -103,6 +121,8 @@ Sometimes governments make ambiguous funding announcements (where it is not clea
 
 ### H2 - Testing Policy
 
+- If there is a policy which says that everyone who is symptomatic could walk in and get a test, but this could not be met if everyone symptomatic walked into the clinic at once, this suggests there is not the capacity to meet this. While we are coding government policies, capacity issues affect implementation. This therefore requires judgment, about how aspirational a policy is versus how practical it is to see if it is able to meet its propositions. Therefore as well as the policy announced, evidence should be provided demonstrating that there is capacity on the ground to meet this testing policy, mirroring the capacity logic used for H3.
```

Comment:
This seems a bit long, can we condense this a bit more.
Also: I would think our guidance on capacity would be the other way around. We take the government at the word, unless there is evidence (eg. press reports of people not being able to get tested) to suggest there is low capacity.

Finally: this is also worded as if the intended audience is coders, not data users.